src/node.cc

```diff
@@ -1813,6 +1813,14 @@ bool AllowWasmCodeGenerationCallback(
 Isolate* NewIsolate(ArrayBufferAllocator* allocator, uv_loop_t* event_loop) {
   Isolate::CreateParams params;
   params.array_buffer_allocator = allocator;
+
+  double totalMemory = uv_get_total_memory();
+  if (totalMemory > 0) {
```

Comment:
Two questions:
1) is it safe to continue if `totalMemory > 0` i.e. `uv_get_total_memory` returned an error?
2) semver-ity (in continuation of discussion thread). Maybe this should be `major` and so we can think less about nuanced implications?

Reply:
P.S. does this have priority over CLI parameters (e.g. `--max-old-space-size`)?

Reply:
> is it safe to continue if totalMemory < 0 i.e. uv_get_total_memory returned an error?

I believe so. For whatever reason (security policy?) we don't know how much memory is available, but that doesn't imply memory allocation is going to fail.